src/cpu/aarch64/cpu_isa_traits.hpp

```diff
@@ -102,6 +102,11 @@ cpu_isa_t DNNL_API get_max_cpu_isa_mask(bool soft = false);
 status_t set_max_cpu_isa(dnnl_cpu_isa_t isa);
 dnnl_cpu_isa_t get_effective_cpu_isa();
 
+// If isa is a superset of sve_128, return sve_128, else return isa
+constexpr cpu_isa_t to_vla_sve(const cpu_isa_t isa) {
+    return (cpu_isa_t)(isa & sve_128);
```

Comment:
I'm not so sure it returns `isa` if `isa` doesn't contain `sve_128`. I'd expect it returning `0` cast to `cpu_isa_t` which might not be expected.